addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiActorGatewayHandler.java

```diff
@@ -126,17 +130,22 @@ void execute(ChannelUID channelUID, Command command) {
 
     @Override
     void parseCommand(String command, JsonObject data) {
-        if (command.equals("report") || command.equals("heartbeat") || command.equals("write_ack")) {
+        if (command.equals("report")) {
```

Comment:
Because of the added readAck and writeAck methods, this override is probably not required anymore, right?